docs/binary.md

```diff
@@ -5,7 +5,12 @@ nav_order: 2
 ---
 
 # Binary numbers
-{: .no_toc }
```

Comment:
Also, since the name of the page is _Binary Numbers_ itself, I don't think that we should be including it in the TOC. Becomes a little redundant. Especially since the corresponding index number for this heading in the TOC is _1_, making all other headings sub-headings to this page. Ideally, the subheadings, say _Introduction_, _Binary Counting_ etc. should be numbered as _1_, _2_ and so on respectively. 

TLDR; the **_.no_toc_** should be present in my opinion.

Reply:
updated